components/ListItems/ListItem.module.css

```
@@ -1,7 +1,7 @@
 .listItem {
   background: var(--list-item-background);
   color: var(--list-item-color);
-  border-top: 1px solid var(--list-item-border-color);
+  border-top: 1px solid #ddd;
```

```suggestion
  border-top: 1px solid var(--list-item-border-color);
```
Return to the var and then change the var itself